CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.1.0-nullsafety.5
+
+* Use 2.12.0-0 lower bound and 3.0.0 upper bound SDK to work around pub
```

Comment:
we probably don't need to mention the pub limitations part in all of these

Reply:
We should say _something_. How would you phrase the impact of this change?

Reply:
Maybe "Update sdk constraints to `>=2.12.0-0 <3.0.0` based on beta release guidelines"

Reply:
Done.